doc/schemas/renepay.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "payment_preimage",
+    "payment_hash",
+    "created_at",
+    "parts",
+    "amount_msat",
+    "amount_sent_msat",
+    "status"
+  ],
+  "properties": {
+    "payment_preimage": {
+      "type": "secret",
+      "description": "the proof of payment: SHA256 of this **payment_hash**",
+      "added": "v23.08"
```

Comment:
Minor change: only put "added" at the top level of each file, since the entire call was introduced, rather than individual fields.

Reply:
Got it.